impy/common.py

```diff
@@ -404,6 +487,14 @@ def set_event_kinematics(self, evtkin):
         """
         pass
 
+    @property
+    def event_kinematics(self):
+        return self._curr_event_kin
+    
+    @event_kinematics.setter
+    def event_kinematics(self, evtkin):
+        self.set_event_kinematics(evtkin)
```

Comment:
Make `set_event_kinematics` private, so that there is only "one obvious way" to set it (see Zen of Python), via the property.

Reply:
Ok